infra/scripts/validate-version-consistency.sh

```diff
@@ -123,7 +125,7 @@ for i in "${files_to_validate_version[@]}"; do
   echo "Testing whether versions are correctly set within file: $FILE_PATH"
   ACTUAL_OCCURRENCES=$(grep -c -P "\bv?$VERSION\b" "$FILE_PATH" || true)
 
-  if [ "${ACTUAL_OCCURRENCES}" -eq "${EXPECTED_OCCURRENCES}" ]; then
+  if [ "${ACTUAL_OCCURRENCES}" -ge "${EXPECTED_OCCURRENCES}" ]; then
```

Comment:
Actual occurrences may be greater that expected occurrences when both pom.xml and tag versions the same. This is required as lint flags any occurrence of the version.